### ISSUE TITLE
Color Contrast on Landing Page

### ISSUE THREAD

**Issue by mgifford** [mozilla/hubs#2241]
**Description**
There isn't sufficient color contrast footer links
https://contrast-ratio.com/#rgb%28128%2C%20128%2C%20128%29-on-white

**To Reproduce**
1. Go to https://hubs.mozilla.com/#/
2. Run WAVE Toolbar
3. See error

<img width="978" alt="Screen Shot of home page" src="https://user-images.githubusercontent.com/116832/76965275-67c07400-68fa-11ea-8aa0-dab09c11bbf8.png">

**Expected behavior**
It should have sufficient contrast.

**Comment by harshil1712** [comment]
I would like to work on this. 
How about using #676767 9https://contrast-ratio.com/#%23676767-on-white), it solves the issues and is a shade of grey.

**Comment by brianpeiris** [comment]
Sounds good to me @harshil1712

**Comment by brianpeiris** [comment]
Closed in #2386. Thanks @harshil1712!
